configuration/camel2bom.fuse7.properties

```diff
@@ -8,3 +8,4 @@
 2.21.0.fuse-750033-redhat-00001=7.5.0.fuse-750029-redhat-00002
 2.21.0.fuse-760027-redhat-00001=7.6.0.fuse-760027-redhat-00001
 2.21.0.fuse-770013-redhat-00001=7.7.0.fuse-770012-redhat-00003
+2.23.2.fuse-780036-redhat-00001=7.8.0.fuse-sb2-780038-redhat-00001
```

Comment:
this one was for Springboot 1, so I think there should not be a new version here

Reply:
ok, I will remove it then.